shared/src/packages/surveys/libs/validation-schemas/survey-request.validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import joi from 'joi';
+
+import { SurveyValidationMessage } from '../enums/enums.js';
+import { type SurveyRequestDto } from '../types/types.js';
+
+const SurveyValidationSchema = joi.object<SurveyRequestDto, true>({
+  userId: joi.number().integer().required().messages({
+    'any.required': SurveyValidationMessage.USER_ID_REQUIRED,
```

Comment:
Add string empty too, so that we don't allow to pass empty string
```suggestion
    'any.required': SurveyValidationMessage.USER_ID_REQUIRED,
    'string.empty': SurveyValidationMessage.USER_ID_REQUIRED,
```